src/modeler/modeler/Modeler.cpp

```diff
@@ -143,11 +143,16 @@ void Modeler::run() const
       parameters.lastTimeStep,  // global = local
       0};
 
+    std::unique_ptr<BendersDecomposition> bendersDecomposition;
+    if (data.resolutionMode == Antares::Modeler::ResolutionMode::BENDERS_DECOMPOSITION)
+    {
+        bendersDecomposition = std::make_unique<BendersDecomposition>();
+    }
+
     // Sub problem
-    auto masterAndSubPbvars = std::make_unique<MasterAndSubPbVariables>();
     std::string pbId = "1-1";
-    masterAndSubPbvars->setProblemIdentifier(pbId);
-
+    bendersDecomposition->setCurrentProblemId(pbId);
+    
```

Comment:
If `data.resolutionMode != Antares::Modeler::ResolutionMode::BENDERS_DECOMPOSITION`, then `bendersDecomposition` points to `nullptr` (`std::unique_ptr`'s default constructor does that) and this line triggers a segfault.